src/main/java/seedu/address/logic/Messages.java

```diff
@@ -31,6 +31,7 @@ public class Messages {
                 "Multiple values specified for the following single-valued field(s): ";
     public static final String MESSAGE_ASSIGNMENT_GRADED = "The assignment has already been graded";
     public static final String MESSAGE_ASSIGNMENT_UNGRADED = "The assignment has not been graded";
+    public static final String MESSAGE_INVALID_TAB_INDEX = "The inputted index is out of valid range";
```

Comment:
I've nvr seen the word `inputted` before. Did a search, and wow there are 2 past tenses for input.
input and inputted are accepted.